pkg/core/blockchain.go

```diff
@@ -1327,27 +1327,27 @@ func (bc *Blockchain) notificationDispatcher() {
 		// These are just sets of subscribers, though modelled as maps
 		// for ease of management (not a lot of subscriptions is really
 		// expected, but maps are convenient for adding/deleting elements).
-		blockFeed        = make(map[chan *block.Block]bool)
-		headerFeed       = make(map[chan *block.Header]bool)
-		txFeed           = make(map[chan *transaction.Transaction]bool)
-		notificationFeed = make(map[chan *state.ContainedNotificationEvent]bool)
-		executionFeed    = make(map[chan *state.AppExecResult]bool)
+		blockFeed        = make(map[chan<- *block.Block]bool)
+		headerFeed       = make(map[chan<- *block.Header]bool)
+		txFeed           = make(map[chan<- *transaction.Transaction]bool)
+		notificationFeed = make(map[chan<- *state.ContainedNotificationEvent]bool)
+		executionFeed    = make(map[chan<- *state.AppExecResult]bool)
```

Comment:
Something has happened with commit indent.

Reply:
Adjust commit and PR description: `chan<-` is a write-only (or more precise send-only) channel.

Reply:
Change commit title to be more precise, I like the way how PR title looks. blockchain: make SubscribeFor* channel arguments receive-only.

Reply:
Link the original issue.